book/Makefile

```
@@ -20,7 +20,7 @@ strict:
 	cd website && jupyter-book build --html --strict
 
 ci:
-	cd website && jupyter-book build --html --ci
+	cd website && jupyter-book build --html --ci --strict
```

```suggestion
	cd website && jupyter-book build --html --ci --strict --link-checker=false
```
not sure if this is the right format, let's see...